packages/ubus-tmate/files/usr/lib/lua/tmate.lua

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env lua
+--[[ Copyright (C) 2013-2020 LibreMesh.org
+  This is free software, licensed under the GNU AFFERO GENERAL PUBLIC LICENSE Version 3
+
+]]--
+
+
+local TMATE_SOCK = "/tmp/tmate.sock"
+local TMATE_CONFIG = "/etc/tmate/tmate.conf"
+
+local tmate = {}
+
+function tmate.cmd_as_str(cmd)
+   final_cmd = "tmate -f "..TMATE_CONFIG.." -S "..TMATE_SOCK.." "..cmd
+   return utils.unsafe_shell(final_cmd)
```

Comment:
utils (lime.utils) it's not imported in this file.

Reply:
Indeed and it is working because lime.utils is "installed" (by mistake) in the global namespace. There are one or two other modules that have the same issue.